python/lsst/ip/isr/isrTask.py

```diff
@@ -789,6 +804,27 @@ def overscanCorrection(self, exposure, amp):
             collapseRej=self.config.overscanRej,
         )
 
+    def addDistortionModel(self, exposure, camera):
+        """!Update the WCS in exposure with a distortion model based on camera geometry
```

Comment:
A tad more detail would be nice here (i.e. expand a bit on the "based on camera geometry" statement)